src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -103,19 +104,27 @@
             TokenTypes.DOT,
             TokenTypes.TYPECAST,
             TokenTypes.ARRAY_DECLARATOR,
-            TokenTypes.GENERIC_START,
-            TokenTypes.GENERIC_END,
+            TokenTypes.INDEX_OP,
         };
     }
 
     @Override
-    public int[] getRequiredTokens() {
-        return ArrayUtils.EMPTY_INT_ARRAY;
-    }
-
-    @Override
```

Comment:
@mkordas , what do you expect there ? If it is just counting of metrics  - please switch it off. We need notification only if method is more then approved limit.